### DISCUSSION TITLE
Code highlighting for yaml

### DISCUSSION THREAD

**Discussion by Dpananos** [quarto-dev/quarto-cli#3256] [Q&A] [answered]
I'm writing some technical documentation about a library which makes heavy use of yaml files.

I would really love for the example yaml I write to be highlighted similar to the way my python code is highlighted.

As of now, I'm using ~~~ to show my yaml files.  This looks like

![image](https://user-images.githubusercontent.com/20362941/200903903-21caaf8d-01b0-4634-a278-222f7e934330.png)

Is there a way to highlight yaml code?

**Comment by cscheid** [comment, marked as answer]
````
```yaml
key: value
hello: world
```
````

should work
